Makefile

```diff
@@ -117,5 +117,20 @@ gen:
 
 .PHONY: clean
 clean:
-	rm packetbeat || true
-	rm -r packetbeat-$(VERSION) || true
+	-rm packetbeat
+	-rm packetbeat.test
+	-rm -r packetbeat-$(VERSION)
+	-rm -r coverage
+
+# Generates packetbeat.test coverage testing binary
+test-binary:
+	$(GODEP) go test -c -cover -covermode=count -coverpkg ./...
```

Comment:
I suggest to name the target packetbeat.test and make it depend on the $(GOFILES) just like the Packetbeat one. Alternatively, declare it as .PHONY, but I think first version is  better.